File: Dev/asd_cs/Particular/GC.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace asd.Particular
{
	class GC
	{
		public static void SuppressFinalize(object obj)
		{
			System.GC.SuppressFinalize(obj);
		}

		public static void CollectAll()
		{
			System.GC.Collect();
			System.GC.WaitForPendingFinalizers();
			System.GC.Collect();
		}
	}
}

File: Dev/asd_java/Particular/GC.java
package asd.Particular;

public class GC
{
	public static void SuppressFinalize(Object obj)
	{
		// 対応するメソッドがない
		// ファイナライザが必ず呼ばれる
	}

	public static void CollectAll()
	{
		java.lang.System.gc();
	}
}
